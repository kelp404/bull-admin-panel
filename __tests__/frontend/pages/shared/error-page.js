const React = require('react');
const ReactTestRenderer = require('react-test-renderer');
const ErrorPage = require('../../../../lib/frontend/src/pages/shared/error-page');

afterEach(() => jest.restoreAllMocks());

test('render error page with error information', () => {
  const error = new Error('fake error');
  const render = ReactTestRenderer.create(<ErrorPage error={error}/>);

  expect(render.toJSON()).toMatchSnapshot();
});

test('render error page with string', () => {
  const error = 'fake error';
  const render = ReactTestRenderer.create(<ErrorPage error={error}/>);

  expect(render.toJSON()).toMatchSnapshot();
});
