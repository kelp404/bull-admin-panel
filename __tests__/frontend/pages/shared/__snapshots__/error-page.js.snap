// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render error page with error information 1`] = `
<p
  className="text-center h2"
>
  fake error
</p>
`;

exports[`render error page with string 1`] = `
<p
  className="text-center h2"
>
  fake error
</p>
`;
